public/js/controllers/scroll.js

```diff
@@ -1,7 +1,7 @@
 angular.module('mean.system')
   .controller('ScrollController', ['$scope', '$location', '$anchorScroll',
-    function ($scope, $location, $anchorScroll) {
-      $scope.goPlay = function () {
+    ($scope, $location, $anchorScroll) => {
```

Comment:
Change back to function definition not arrow function

Reply:
I didn't change this

Reply:
Fix it still